spec.md

```diff
@@ -296,22 +305,22 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id/las
 | Status Code  |  Description |
 |---|---|
 | 200 OK |  The expected response body is below. |
-|  410 GONE | Appropriate only for asynchronous delete operations. The platform should consider this response a success and remove the resource from its database. The expected response body is <code>{}</code>. Returning this while the platform is polling for create or update operations should be interpreted as an invalid response and the platform should continue polling.  |
+|  410 GONE | Appropriate only for asynchronous delete operations. The platform can consider this response a success and remove the resource from its database. The expected response body is <code>{}</code>. Returning this while the platform is polling for create or update operations SHOULD be interpreted as an invalid response and the platform SHOULD continue polling.  |
```

Comment:
`can consider` seems a little ambiguous, given that the sentence has technical statements about what the platform should do.  Is this a SHOULD or a MUST or out of scope?